backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CreateVolumeContainerCommand.java

```diff
@@ -85,6 +86,10 @@ private VolumeType getType() {
             return getParameters().getVolumeType();
         }
 
+        if (getParameters().getBackup() == DiskBackup.Incremental) {
+            return getParameters().getVolumeType();
+        }
+
```

Comment:
I think it's better to add a comment before that one, mentioning we want to keep the disk as preallocated instead of thin provitioned.

And, it can be joined to the cond. before that:
```
if (getParentParameters() != null && getParentParameters().getParentCommand() == ActionType.ConvertDisk
          || getParameters().getBackup() == DiskBackup.Incremental) {

Reply:
I suggest to remove it entirely (lines 85-87), it is no longer needed with this change, can be done separately

Reply:
ACK